### PR TITLE
Clarifications relating to system libraries, capability jumps; typo correction.

### PR DESCRIPTION
**Pull request by rwatson**
Addresses #6.

### CODE REVIEW

File: src/performance-methodology/baseline-and-comparison-framework.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ including its system libraries.
 The SPECint benchmark suite is compiled with -O3. SPEC is statically linked in
 all presented results.
 
-By default, CheriBSD 22.12 ships with unoptimized third-party libraries (compiled with -O0), to improve debugging in the initial release, and those would not be suitable for benchmarking; we intend to provide packages built with optimizations enabled in the future. SPECint does not have external software dependencies beyond the system library, and so our work is not affected. If we were dependent on third-party packages, as might be true of other benchmark suites, we would need to investigate optimization settings for those packages. 
+By default, CheriBSD 22.12 ships with unoptimized third-party libraries (compiled with -O0), to improve debugging in the initial release, and those would not be suitable for benchmarking; we intend to provide packages built with optimizations enabled in the future. SPECint does not have external software dependencies beyond `libc`, and so our work is not affected. If we were dependent on third-party packages, as might be true of other benchmark suites, we would need to investigate optimization settings for those packages.
```
Comment (arichardson) [inline src/performance-methodology/baseline-and-comparison-framework.md]: there is also libunwind and libc++.

Comment (jrtc27) [reply src/performance-methodology/baseline-and-comparison-framework.md]: "beyond libraries in the base system"?

Comment (jrtc27) [reply src/performance-methodology/baseline-and-comparison-framework.md]: > there is also libunwind and libc++.

and libcxxrt, libm, libgcc_s (compiler-rt)

**Review by arichardson — APPROVED** [review]

File: src/performance-analysis-spec/specific-hardware-and-software.md [hunk 2]
```diff
@@ -85,8 +85,8 @@ GOT entries, including those used for the PLT, remain 64 bit.
 ## P128 (w/ data-dependency fix) (w/ larger store queue)
 
 This configuration is identical to the P128 Forced-GOT configuration except
-that access to globals is sometimes performed via PCC, bypas
-ing GOT entries in configurations where aarch64 would offer the same choice
+that access to globals is sometimes performed via PCC, bypassing GOT entries
```
Comment (jrtc27) [inline src/performance-analysis-spec/specific-hardware-and-software.md]:
```suggestion
that access to globals is sometimes performed via PC, bypassing GOT entries
```
maybe? It’s PC-relative with an integer base so not using PCC but PC via DDC

Comment (rwatson) [reply src/performance-analysis-spec/specific-hardware-and-software.md]: Oh, I see what you mean, in the P128 case. Yes.

**Pull request merged**